static/src/javascripts/projects/commercial/modules/prebid/slot-config.js

```diff
@@ -92,9 +98,17 @@ const getSlots = (contentType: string): Array<PrebidSlot> => {
         },
     ];
 
+    const mobileStickySlot: PrebidSlot = {
+        key: 'mobile-sticky',
+        sizes: [[320, 50]],
+    };
+
     switch (getBreakpointKey()) {
         case 'M':
-            return commonSlots.concat(mobileSlots);
+            return isInNA() &&
```

Comment:
Does prebid require this isInNA check? The mobile sticky is already restricted to rendering only in NA so if that's not the case then I think you can drop the check here.

Reply:
Pre-bid runs independently of mobile sticky and doesn't know the conditions of when the mobile sticky slot is shown

Reply:
👍